src/pilot/install-director.sh

```diff
@@ -194,7 +216,17 @@ echo "## Done."
 
 echo 
 echo "## Customizing the overcloud image & uploading images"
-run_command "~/pilot/customize_image.sh ${subscription_manager_user} ${subscription_manager_pass} ${subcription_manager_poolid} ${proxy}"
+if [ ! -z "${satellite_hostname}" ]; then
+    run_command "~/pilot/customize_image.sh --satellite_hostname ${satellite_hostname} \
+                --satellite_org ${satellite_org} \
+                --satellite_key ${satellite_key} \
+                --proxy ${proxy}"
+                  
+elif [ ! -z "${subscription_manager_user}" ];then
```

Comment:
might want to change the above to just an "else:" or add an else below that logs a fatal error and exits.  if it's not satellite or subscription manager, what will happen?